components/commons/basebutton-coretokens.css

```diff
@@ -47,7 +47,8 @@ governing permissions and limitations under the License.
 
   text-decoration: none;
   font-family: var(--mod-font-family-base, var(--spectrum-font-family-base));
-  line-height: var(--mod-line-height-small, var(--spectrum-line-height-small)); 
+
+  line-height: var(--mod-line-height-100, var(--spectrum-line-height-100));
```

Comment:
@pfulton Is the idea that mod vars mimic the naming of the variables they fallback to? Should this remain at --mod*-small or be migrated to the new --mod*-100

Reply:
Honestly, using `--spectrum-line-height-100` is _probably_ a bit brittle here and we should look to refactor to something more abstract than referring to the "global", but this is also part of this shared `commons` file and we don't have a single component name we could reference or a single selector we could define the abstraction under.

Reply:
> Is the idea that mod vars mimic the naming of the variables they fallback to?

But yes! That's correct 😄 - the new token is the `-100` naming convention.